common/src/main/scala/io/prediction/configuration/SSLConfiguration.scala

```diff
@@ -0,0 +1,52 @@
+package io.prediction.configuration
+
+/**
+  * Created by ykhodorkovsky on 2/26/16.
+  */
+
+import java.io.FileInputStream
+import java.io.File
+import java.security.KeyStore
+import javax.net.ssl.{KeyManagerFactory, SSLContext, TrustManagerFactory}
+
+import com.typesafe.config.ConfigFactory
+import spray.io.ServerSSLEngineProvider
+
+trait SSLConfiguration {
+
+  private val serverConfig = ConfigFactory.load("server.conf")
+
+  private val keyStoreResource = serverConfig.getString("io.prediction.server.ssl-keystore-resource")
+  private val password = serverConfig.getString("io.prediction.server.ssl-keystore-pass")
+  private val keyAlias = serverConfig.getString("io.prediction.server.ssl-key-alias")
+
+  private val keyStore = {
+
+    //Loading keystore from specified file
+    val clientStore = KeyStore.getInstance("JKS")
+    val inputStream = new FileInputStream(keyStoreResource)
```

Comment:
Just tested the latest code. Sorry that I did not realize this earlier. This also requires a special treatment. Guess we need to do something like `getClass().getClassLoader().getResource(keyStoreResource).getFile())`.